search-app/src/cmr/search/services/query_service.clj

```diff
@@ -427,6 +430,45 @@
      :hits (:hits results)
      :result-format result-format}))
 
+(defn get-deleted-granules
+  "Executes elasticsearch searches to find collections that are deleted from a given revision-date.
+   This only finds granules that are truly deleted and not searchable.
+   granules that are deleted, then later ingested again are not included in the result."
+
+  [context params]
+  (pv/validate-deleted-granules-params params)
+  (let [start-time (System/currentTimeMillis)
+        {:keys [parent-collection-id provider revision-date result-format]} (common-params/sanitize-params params)
+        filters [{:range {:revision-date {:gte revision-date}}}]
```

Comment:
You should be able to use existing code to help build up a query model and then call execute query so it follows through most of the same paths in the code. I'm wondering if the deleted-collections endpoint might be a good example of how we perform elasticsearch queries.

Reply:
The existing query building framework requires a concept-type. There is no concept type for this query.